oras/auth/token.py

```diff
@@ -156,6 +160,7 @@ def request_anonymous_token(self, h: auth_utils.authHeader) -> bool:
         # From https://docs.docker.com/registry/spec/auth/token/ section
         # We can get token OR access_token OR both (when both they are identical)
         data = response.json()
+        print("data: %s" % data)
```

Comment:
This should probably be debug or removed for the PR.

Reply:
Yeah dubbing